cmd/wallet-sdk-gomobile/openid4vp/interaction.go

```diff
@@ -146,13 +148,10 @@ func (o *Interaction) GetQuery() ([]byte, error) {
 }
 
 // VerifierDisplayData returns display information about verifier.
-func (o *Interaction) VerifierDisplayData() (*VerifierDisplayData, error) {
-	displayData, err := o.goAPIOpenID4VP.VerifierDisplayData()
-	if err != nil {
-		return nil, wrapper.ToMobileErrorWithTrace(err, o.oTel)
-	}
+func (o *Interaction) VerifierDisplayData() *VerifierDisplayData {
```

Comment:
This is breaking change. You can keep old signature with error '(*VerifierDisplayData, error)'. Just always return nil. Or add ! to PR title.

Reply:
It doesn't actually break any existing code (just an IDE warning/complaint for iOS)